colorcoder/Main.java

```diff
@@ -1,120 +1,28 @@
 package colorcoder;
 
-public class Main {
-    enum MajorColor {
```

Comment:
Same Name (Main) used for Class/Method
Main Class holds too many responsibilities